src/main/java/org/c4sg/service/impl/ProjectServiceImpl.java

```diff
@@ -234,12 +234,15 @@ private void apply(User user, Project project) {
         		String orgEmail = userList.get(0).getEmail();
         	
         		if (!StringUtils.isEmpty(orgEmail)) {
+        			asyncEmailService.send(FROM_EMAIL, orgEmail, SUBJECT_ORGANIZATION, BODY_ORGANIZATION);
+        			/* javax.mail.AuthenticationFailedException on Heroku server
```

Comment:
This might have something to do with the change in reading the mail properties from `application.properties` now